### DISCUSSION TITLE
i cannot deploy on Vercel

### DISCUSSION THREAD

**Discussion by anhnhantn97** [morethanmin/morethan-log#106] [Q&A] [answered]
This is my issue
![image](https://user-images.githubusercontent.com/89826873/218307129-b6248fba-fa25-42f2-a3b6-1c18a0270805.png)
![image](https://user-images.githubusercontent.com/89826873/218307139-5f0f1329-23f9-4bb6-9f82-dac1214842d1.png)
And here's the config I have changed
![image](https://user-images.githubusercontent.com/89826873/218307196-05316aad-5a14-487b-b591-c1fb4f6ec7f6.png)

**Comment by anhnhantn97** [comment, marked as answer]
The problem are solved.

**Reply by 0ops5** [reply]
how ?

**Comment by anhnhantn97** [comment]
I wanna change logo from String to Image  
![image](https://user-images.githubusercontent.com/89826873/218313641-7c743f4f-671e-44e5-b6cf-21e51a3529a9.png)

help me please
